libs/core-wrappers/src/lib/Wizard/Wizard.tsx

```diff
@@ -0,0 +1,194 @@
+    [goToNamedStep]
+  );
+  return (
+    <div className={styles.summary}>
+      <h3>Summary</h3>
+      {steps.map((step) => (
+        <div
+          className={styles['step-summary']}
+          key={`wizard-summary-${step.id}`}
+        >
+          <div className={styles.name}>
+            <b>{step.name}</b>
+            <Button
+              color="link"
+              onClick={() => editCallback(step.id)}
+              className={styles.edit}
+            >
+              edit
+            </Button>
+          </div>
+          <div className={styles.content}>{step.summary}</div>
+        </div>
+      ))}
+    </div>
+  );
+}
+
+type StepContainerProps<T> = {
+  step: WizardStep<T>;
+  formSubmit: (values: Partial<T>) => void;
+} & Partial<StepWizardChildProps>;
+
+function StepContainer<T>({ step, formSubmit }: StepContainerProps<T>) {
+  const { validationSchema, initialValues, validate } = step;
+  return (
+    <Formik
+      validationSchema={validationSchema}
+      initialValues={initialValues}
+      validate={validate}
+      onSubmit={formSubmit}
+      enableReinitialize={true}
+    >
+      <Form>
+        <div className={styles.step}>
+          {step.render}
+          <WizardNavigation />
+        </div>
+      </Form>
+    </Formik>
+  );
+}
+
+type WizardProps<T> = {
+  steps: Array<WizardStep<T>>;
+  memo?: any;
+  formSubmit: (values: Partial<T>) => void;
+};
+
+function Wizard<T>({ steps, memo, formSubmit }: WizardProps<T>) {
+  const [stepWizardProps, setStepWizardProps] = useState<
+    Partial<StepWizardChildProps>
+  >({});
+
+  const instanceCallback = useCallback(
+    (props: Partial<StepWizardChildProps>) => {
+      setStepWizardProps({
+        currentStep: 1,
+        totalSteps: steps.length,
+        ...props,
+      });
+    },
+    [setStepWizardProps, steps]
+  );
+
+  const stepChangeCallback = useCallback(
+    ({ activeStep }: { previousStep: number; activeStep: number }) => {
+      setStepWizardProps({
+        ...stepWizardProps,
+        currentStep: activeStep,
+      });
+    },
+    [setStepWizardProps, stepWizardProps]
+  );
+
+  const { goToStep } = stepWizardProps;
+
+  useEffect(
+    () => {
+      goToStep && goToStep(1);
+    },
+    /* eslint-disable-next-line */
+    [memo]
+  );
+
+  return (
+    <WizardContext.Provider value={stepWizardProps}>
+      <div className={styles.container}>
+        <StepWizard
+          instance={instanceCallback}
+          className={styles.steps}
+          onStepChange={stepChangeCallback}
+          transitions={{}}
+        >
+          {steps.map((step) => (
+            <StepContainer<T>
+              step={step}
+              key={`wizard-step-${step.id}`}
+              stepName={step.id}
+              formSubmit={formSubmit}
+            />
+          ))}
+        </StepWizard>
+        <WizardSummary steps={steps} {...stepWizardProps} />
+      </div>
+    </WizardContext.Provider>
+  );
+}
+
+export default Wizard;
```

Comment:
**Wizard**
If I try to Continue past Step 2, the error for required field does not appear. **If** I put in a value, **then** remove a value, then the error for required field appears.

https://user-images.githubusercontent.com/62723358/175393646-6ab488a9-89ee-4c33-92cf-7e0697a1e9d3.mov